### ISSUE TITLE
Need Distance in miles

### ISSUE THREAD

**Issue by MadhavNasit** [bramus/react-native-maps-directions#163]
I need distance between origin and destination in miles instead of km any props or any way that i can achieve that?

**Comment by Sheharyar566** [comment]
@MadhavNasit basic maths is the way to go. To get the distance in miles rather than kilometers, simply divide the km distance by 1.609 (just saw the formula sort of thingie on Google 😅)

**Comment by MadhavNasit** [comment]
> @MadhavNasit basic maths is the way to go. To get the distance in miles rather than kilometers, simply divide the km distance by 1.609 (just saw the formula sort of thingie on Google 😅)

Yes but prop would be great for package. They are converting miles from api into km, so they can return miles as it is too.
